Created AuntJudysXXX scraper

echo6ix opened the pull request.

Maista6969 commented on scrapers/AuntJudysXXX.yml:
```diff
@@ -0,0 +1,24 @@
+    scene:
+      Studio:
+        Name:
+          fixed: Aunt Judy's XXX
```
I suggest replacing this to support Aunt Judy's as well:
```
Studio:
  Name:
    selector: //base/@href
    postProcess:
      - replace:
          - regex: ^https?://(?:www\.)?(.+)\.com/.*$
            with: $1
      - map:
          auntjudys: Aunt Judy's
          auntjudysxxx: Aunt Judy's XXX
```

Maista6969 reviewed (approved):
Noticed that this site has the same structure as `auntjudys.com` so I suggest that you include this as well 😄

echo6ix commented:
> Noticed that this site has the same structure as `auntjudys.com` so I suggest that you include this as well 😄

Good idea.

- Now scrapes auntjudys.com using @Maista6969's suggestions
- Renamed yaml to `AuntJudys.yml` (becausee AuntJudyXXX is derivative of AuntJudys)
- Updated scrapers list to reflect the above changes

Read for review.
